src/server_manager/ui_components/outline-server-view.html

```diff
@@ -604,12 +564,6 @@ <h3>[[accessKeyRows.length]]</h3>
       </iron-pages>
     </div>
 
-    <outline-help-bubble id="getConnectedHelpBubble" vertical-align="bottom" horizontal-align="right">
-      <img src="images/connect-tip-2x.png" />
```

Comment:
If we're dropping the help bubble, I don't think this image is used elsewhere.

Reply:
Done